src-built-in/components/searchMenu/src/components/ProviderList.jsx

```diff
@@ -31,13 +31,7 @@ export default class ProviderList extends React.Component {
 			<div className="provider">
 				<div className="searchHeader">
 					{providerInfo.provider.displayName}
-
 				</div>
-
```

Comment:
@DanNicolai this looks like bad merge? I think that I added these lines of code into the seed.